build/scripts/build-utils.sh

```diff
@@ -28,16 +28,16 @@ get_version_core()
 #   get_package_version System.Console
 get_package_version() 
 {
-    local repoDir=$(get_repo_dir)
-    local version=$(get_version_core $1 ${repoDir}/build/Targets/Packages.props)
-    echo $version
+    local repoDir="$(get_repo_dir)"
```

Comment:
I'm not sure if things of the form `var=$(command)` need quoting (quick testing says I don't think so?), but it can't hurt, I guess. (Lots of places in this PR do this, just pointing it out at an example place)

Reply:
It's a strange case: if we were applying a command (like in `cat $(get some files)`), we would *definitely* want quoting. Here, likely not strictly necessary. But it sets a better code style for the future, I should think.